app/helpers/application_helper.rb

```diff
@@ -55,10 +55,10 @@ def current_locale_logo
 
   def contact_us_path
     if I18n.locale == :'fr-CA'
-      'https://portagenetwork.ca/fr/contactez-nous/'
+      'fr/contactez-nous'
     else
       # Handling :'en-CA' locale
-      'https://portagenetwork.ca/contact-us/'
+      'contact-us'
```

Comment:
Suppose we are not mentioning the full path! if it is in https://uat.library.ualberta.ca:3000/! will this work?